source/install/desktop-app-install.rst

```diff
@@ -18,138 +18,187 @@ Desktop application install guides
+    3. Run Mattermost as a desktop app.
 
-The following Group Policies are available:
+    **Arch Linux-based systems**
 
-+----------------------------+-----------------------------------------------------------------------------+----------------------+
-| Group Policy               | Description                                                                 | Required Version     |
-+============================+=============================================================================+======================+
-| Enable Server Management   | If disabled, management of servers in the app settings are disabled.        | 4.3 or later         |
-+----------------------------+-----------------------------------------------------------------------------+----------------------+
-| Default Server List        | Define one or more default, permanent servers.                              | 4.3 or later         |
-+----------------------------+-----------------------------------------------------------------------------+----------------------+
+    To install the Desktop client on Arch Linux, see the `Mattermost page <https://wiki.archlinux.org/index.php/Mattermost>`__ on the Arch Linux wiki.
 
-macOS 10.9+
------------
+    **Snapcraft package**
 
-1. Download the latest version of the Mattermost desktop app:
-  - `Intel systems <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-mac-x64.dmg>`__
-  - `M1 systems <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-mac-m1.dmg>`__ (Beta)
+    A snap is available for systems that have Snapcraft installed. Snapcraft is installed by default on Ubuntu 16.04 and later, but for most other Linux distributions you can install it manually. To install Snapcraft, see `Install snapd <https://snapcraft.io/docs/core/install>`__ on the Snapcraft website.
 
-2. Double-click the download to open the disk image.
+    1. At the command line, execute the following command: ``sudo snap install mattermost-desktop --beta``
 
-3. Drag the Mattermost application to the **Applications** folder.
+    2. Run Mattermost as a desktop app.
 
-`Homebrew <https://brew.sh>`__ users can install with ``brew install --cask mattermost``.
+  .. tab:: Mac 10.9+
 
-Linux
------
+    1. Download the latest version of the Mattermost desktop app:
+      
+      - `Intel systems <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-mac-x64.dmg>`__
+      - `M1 systems <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-mac-m1.dmg>`__ (Beta)
+
+    2. Double-click the download to open the disk image.
+
+    3. Drag the Mattermost application to the **Applications** folder.
+
+    .. tip::
+      
+      `Homebrew <https://brew.sh>`__ users can install with ``brew install --cask mattermost``.
```

Comment:
Would following this step download the latest version of the desktop app?

Reply:
I've never done it this way myself, so I'm not sure if it works, but it is no longer the recommended way.

As of May 16th we'll be in the App Store so it would make sense to direct them there. I'll have to get an updated link for you for that once we're in.

Reply:
Thanks, @devinbinnie. Should we remove this set of instructions from the docs since it's no longer our recommended path?

Reply:
Yes, definitely remove this.

Reply:
Removed. Thanks!